src/components/AssetAccountDetails.tsx

```diff
@@ -22,6 +24,7 @@ type AssetDetailsProps = {
 }
 
 export const AssetAccountDetails = ({ assetId: caip19, accountId }: AssetDetailsProps) => {
+  const unavailable = useAppSelector(state => selectMarketDataUnavailableById(state, caip19))
```

Comment:
My question is: What is unavailable? You can find it on this line but you wouldn't be able to know it above